src/api/requests/withdrawals/v1.ts

```diff
@@ -14,6 +14,33 @@ import { externalError } from "../../../common/error";
 import { logger } from "../../../common/logger";
 import { RequestHandlerService } from "../../../services/request-handler";
 
+const SubmitWithdrawalRequestParamsSchema = Type.Object({
+  chainId: Type.String({
+    description: "The chain id of the allocator",
+  }),
+  depository: Type.String({
+    description: "The depository address of the allocator",
+  }),
+  currency: Type.String({
+    description: "The currency to withdraw",
+  }),
+  amount: Type.String({
+    description: "The amount to withdraw",
+  }),
+  spender: Type.String({
+    description: "The address of the spender",
```

Comment:
Please confirm that is the address on the hub? right?

```suggestion
    description: "The address of the spender on the hub",
```